materialsLibrary/src/cell/cellMaterial.ts

```diff
@@ -263,7 +263,7 @@ export class CellMaterial extends PushMaterial {
                 this._activeEffect.setFloat("pointSize", this.pointSize);
             }
 
-            MaterialHelper.BindEyePosition(effect, scene);
+            Scene.BindEyePosition(effect, scene);

```

Comment:
Why static if it is on scene ? I ll put the comment in scene as well

Reply:
Not static anymore